src/main/java/seedu/address/model/AnimalUserPrefs.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+
+import seedu.address.commons.core.GuiSettings;
+
+/**
+ * Represents User's preferences.
+ */
+public class AnimalUserPrefs implements AnimalReadOnlyUserPrefs {
+
+    private GuiSettings guiSettings = new GuiSettings();
+    private Path catalogFilePath = Paths.get("data" , "catalog.json");
+    /**
+     * Creates a {@code AnimalUserPrefs} with default values.
+     */
+    public AnimalUserPrefs() {}
+
+    /**
+     * Creates a {@code AnimalUserPrefs} with the prefs in {@code userPrefs}.
+     */
+    public AnimalUserPrefs(AnimalReadOnlyUserPrefs userPrefs) {
+        this();
+        resetData(userPrefs);
+    }
```

Comment:
JavaDocs could be clearer in differentiating which constructors are used in which situations